lectures/french_rev.md

```diff
@@ -1,1031 +1,1026 @@
+
+ * raising taxes 
+ * lowering government's non debt service (i.e., non-interest)   expenditures 
+ * lowering its debt service (i.e., interest) costs by rescheduling its debt, i.e., defaulting on on part of its debt
+
+The French constitution and prevailing arrangements had empowered three constituencies to block adjustments to components of the government budget constraint that they cared especially about
+
+* tax payers
+* beneficiaries of government expenditures
+* government creditors (i.e., owners of government bonds)
+
+When the French government had confronted a similar situation around 1720 after King  Louis XIV's
+Wars had left it with a debt crisis, it had sacrificed the interests of  
+government creditors, i.e., by defaulting enough of its debt to bring  reduce interest payments down enough to balance the budget.
+
+Somehow, in 1789, creditors of the French government were more powerful than they had been in 1720.
+
+Therefore, King Louis XVI convened the Estates General together to ask them to redesign the French constitution in a way that would lower government expenditures or increase taxes, thereby
+allowing him to balance the budget while also honoring his promises to creditors of the French government.  
+
+
+
+The King called the Estates General together in an effort to promote the reforms that would
+would bring sustained budget balance.  
+
+{cite}`sargent_velde1995` describe how the French Revolutionaries set out to accomplish that.
+
+
+## Nationalization, Privatization, Debt Reduction 
+
+In 1789, the Revolutionaries quickly reorganized the Estates General  into a National Assembly.
+
+A first piece of business was to address the fiscal crisis, the situation that had motivated the King to convence the Estates General.
+
+
+The Revolutionaries were not socialists or communists.
+
+To the contrary, they respected  private property and knew state-of-the-art economics.  
+
+They knew that to honor government debts, they would have to raise new revenues or reduce expenditures.
+
+A coincidence was that the Catholic Church owned vast income-producing properties.  
+
+Indeed, the capitalized value of those income streams put estimates of the value of church lands at 
+about the same amount as the entire French government debt. 
+
+This coincidence fostered a three step plan for servicing the French government debt
+
+ * nationalize the church lands -- i.e., sequester or confiscate it without paying for it
+ * sell the church lands 
+ * use the proceeds from those sales to service or even retire French government debt
+
+The monetary theory underlying this plan had been set out by Adam Smith in his analysis of what he called **real bills**  in his  1776 book
+**The Wealth of Nations**   {cite}`smith2010wealth`, which many of the revolutionaries had read.
+
+Adam Smith defined a **real bill** as a paper money note that is backed by a claims on a real asset like productive capital or inventories. 
```

Comment:
```suggestion
Adam Smith defined a *real bill* as a paper money note that is backed by a claims on a real asset like productive capital or inventories. 
```

Reply:
I think real bill is also a definition here : )